packages/manager/modules/common-translations/public/translations/form/Messages_fr_FR.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mandatory_fields": "Les champs mentionnés avec un astérisque * sont obligatoires.",
+  "max_chars": "{{ value }} caractères maximum",
+  "min_chars": "{{ value }} caractères minimum",
+  "required_field": "Champ obligatoire",
+  "required_fields": "Champs obligatoire",
+  "error_required_field": "Veuillez compléter ce champ.",
+  "error_required_fields": "Veuillez compléter ces champs.",
```

Comment:
We might want to use the plural/count method from i18next instead

Reply:
good idea for some case, but I think for this one it will be hassle since we will require to pass { count } every time, even for single field